lib/hanami/cli.rb

```diff
@@ -131,6 +131,16 @@ def routes
       end
     end
 
+    desc 'secret', 'Prints a secure hash'
+    long_desc <<-EOS
+    `hanami secret` will generate a secure key to be used by the developer as he wants, e.g. cookie sessions.
```

Comment:
'they' is better than 'it' as a gender neutral pronoun. for example: `as they want` sounds better